src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cookies.cs

```diff
@@ -393,6 +394,130 @@ public async Task GetAsync_ReceiveMultipleSetCookieHeaders_CookieAdded()
             });
         }
 
+        // Default path should be calculated according to https://tools.ietf.org/html/rfc6265#section-5.1.4
+        // When a cookie is being sent without an explicitly defined Path for a URL with URL-Path /path/sub,
+        // the cookie should be added with Path=/path.
+        [Fact]
+        public async Task GetAsync_NoPathDefined_CookieAddedWithDefaultPath()
+        {
+            if (PlatformDetection.IsNetFramework)
```

Comment:
`[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotNetFramework)]` maybe ???

Though, we would have to add `PlatformDetection.IsNotNetFramework` since we don't have the negative version of this condition yet.

This is just a suggestion, feel free to leave as-is.